bin/update-pyright.sh

```diff
@@ -8,8 +8,8 @@ set -euxo pipefail
 (cd ../pyright/packages/browser-pyright && npm run build)
 rm public/workers/pyright-*.js*
```

Comment:
This needs to also delete chunk files.

Reply:
It might be possible to set a webpack config option that will prefix the chunk file names so they're easier to delete here.